built_value_generator/lib/src/value_source_field.dart

```diff
@@ -159,6 +167,11 @@ abstract class ValueSourceField
       result.add('Make field $name public; remove the underscore.');
     }
 
+    if (_suggestedTypes.keys.contains(type)) {
+      result.add('Make field "$name" have type "${_suggestedTypes[type]}". '
+          'The current type, "$type", is not allowed.');
```

Comment:
how about: "The current mutable type..." ?
Might be a better explanation to switch

Reply:
Done, "is not allowed because it is mutable."

Thanks!